<?xml version="1.0"?>
<launch>


<!--Publishes a "fake" robot to camera transform, jsut to make sure we can access tfs-->
<arg name="robot_camera_mount_link_frame_id" default="/ee_link"/>
<!--<arg name="camera_frame_id" default="/kinect_rgb_optical_frame"/>-->
<arg name="camera_frame_id" default="/kinect_link"/>
<arg name="pi/2" value="1.5707963267948966" />
<arg name="pi" value="3.14" />


<!--This is yaw pitch roll (must be changed after the tf_echo)-->
<!--<arg name="static_transform" value="0.0 0 0 pi 0 0" />-->
<arg name="static_transform" value="-0.048, 0.066, 0.182, 0.359, -0.220, -0.131, 0.898" />
<!--rosrun tf tf_echo /ee_link /kinect_link-->

<node pkg="tf" type="static_transform_publisher" name="static_tf_publisher" args="$(arg static_transform) $(arg robot_camera_mount_link_frame_id) $(arg camera_frame_id) 30" />

</launch>	





